site/components/src/main/java/uk/nhs/digital/intranet/components/AnnouncementComponent.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.digital.intranet.components;
+
+import org.hippoecm.hst.component.support.bean.BaseHstComponent;
+import org.hippoecm.hst.container.RequestContextProvider;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+import org.hippoecm.hst.core.component.HstComponentException;
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+import org.hippoecm.hst.core.request.HstRequestContext;
+
+public class AnnouncementComponent extends BaseHstComponent {
+
+    @Override
+    public void doBeforeRender(HstRequest request, HstResponse response) throws HstComponentException {
+        super.doBeforeRender(request, response);
+        HstRequestContext requestContext = RequestContextProvider.get();
+        HippoBean contentBean = requestContext.getContentBean();
+
+        if (contentBean != null) {
+            request.setAttribute("document", contentBean);
+        }
```

Comment:
Can you just use the BaseGaComponent ? Does the same thing + also allows full ga tracking in html fields